packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -1,8 +1,60 @@
 import React, { Component } from 'react';
 import isNil from 'lodash/isNil';
 import classNames from 'classnames';
+import { css } from 'emotion';
 import Scrollbars from 'react-custom-scrollbars';
 
+const getStyles = (page?: boolean) => {
+  return {
+    customScrollbar: css`
+      display: flex;
+      flex-grow: 1;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    trackVertical: css`
+      border-radius: 3px;
+      width: 8px !important;
+      right: ${page ? 0 : 2}px;
+      bottom: 2px;
+    `,
+    trackHorizontal: css`
+      border-radius: 3px;
+      height: 8px !important;
+      right: 2px;
+      bottom: 2px;
+      left: 2px;
+    `,
+    thumbVertical: css`
+      @include gradient-vertical($scrollbarBackground, $scrollbarBackground2);
+      border-radius: 6px;
+      opacity: 0;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    thumbHorizontal: css`
+      @include gradient-horizontal($scrollbarBackground, $scrollbarBackground2);
+      border-radius: 6px;
+      opacity: 0;
+      &:hover {
+        opacity: 0.8;
+        transition: opacity 0.3s ease-in-out;
+      }
+    `,
+    view: css`
+      display: flex;
+      flex-grow: 1;
+      flex-direction: column;
+    `,
+  };
+};
+
+const styles = getStyles();
```

Comment:
this looks like it's called out side the component, and the page argument is never passed from the component props to this

Reply:
oh...my bad.
would fix this right away